Legion.BackEnd/database/migrations/2018_07_17_021708_create_announcements_table.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateAnnouncementsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('announcements', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('title');
+            // Will be able to store up to 64KiB a field with play TEXT field. Should be good enough.
+            // Can go up to 4 GiB if needed with LONGTEXT...
+            $table->text('content');
+            $table->unsignedInteger('user_id')->index();
+            $table->unsignedInteger('last_modified_by')->index();
+            $table->timestamps();
+            // Enable announcement soft deleting for records.
+            $table->softDeletes();
+        });
+    }
```

Comment:
Is there a creation date?  I only see last modified?  Also, is there a created by?  Only need to store user IDs, don't worry about names (you may not have, I can't really tell).

Reply:
I store the user id, and there is a creation and last modified dates, those are packaged in the timestamps() function right above the softDeletes.

Reply:
What if someone other than the create modified it, though?

Reply:
The time still gets updated. I just update the modified_by field or whatever I called it.